### DISCUSSION TITLE
How to Install/Use

### DISCUSSION THREAD

**Discussion by ishmaelcamillo** [jabbey1/UGDownloader#23] [Q&A]
Hi, I am new to the world of IDE's, terminals, and Github projects. How do I access the GUI for this project? I cloned the github project to my local directory and pip installed the requirements but I'm not sure where to go from here as to using the GUI/application. Pls help I am very new to these things (ty).

**Comment by jabbey1** [comment]
The easiest thing to do would be to grab the .exe version- it's on the right side of the main github page under the section 'releases'.
But if for some reason you don't want to do it that way, you'll either run it from the terminal or your IDE. Running from an IDE will vary depending on what IDE you choose, but should be pretty simple. Otherwise, you can open your terminal inside the \src\UGDownloader folder. Then, you should be able to type `python UGDownloader.pyw` to run it

**Reply by ishmaelcamillo** [reply]
Hey, thank you a bunch! Can't wait to learn some new songs through this!

**Comment by rodduarte666** [comment]
Amazing work sir!
Quick question: Could I somehow adapt this to, instead of downloading everything for chosen artists, I could have it read a txt file, and download only the songs that are in there?
